core/ledger/kvledger/txmgmt/privacyenabledstate/test_exports.go

```diff
@@ -178,5 +186,6 @@ func (env *CouchDBCommonStorageTestEnv) Cleanup() {
 	os.RemoveAll(env.redoPath)
 	env.bookkeeperTestEnv.Cleanup()
 	env.provider.Close()
+	env.couchAddress = ""
```

Comment:
this is not needed

Reply:
This one is needed. I don't know how to... show git link here, but back to Init function 
	if env.couchAddress == "" {
		env.couchAddress = env.setupCouch()
	}
we should clean up this. otherwise, 
testenv.Init() will not create anymore, and throw up an error as db closed.